routes/api/users.js

```diff
@@ -61,4 +61,18 @@ router.put("/:username",(request,response) =>{
      response.send({msg:"User deleted",data:newdata});
 })
 
+router.put("/:username/grades",(request,response) =>{
```

Comment:
the route must be clean and descriptive,
`/addGrades/:username`,
always leave the params at the end.